t/t-content-type.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+. "$(dirname "$0")/testlib.sh"
+
+begin_test "content-type: is enabled by default"
+(
+  set -e
+
+  reponame="content-type-enabled-default"
+  setup_remote_repo "$reponame"
+  clone_repo "$reponame" "$reponame"
+
+  git lfs track "*.zip"
+  printf "aaaaaaaaaa" > a.txt
+  zip -j a.zip a.txt
+  rm a.txt
+
+  git add .gitattributes a.zip
+  git commit -m "initial commit"
+  GIT_CURL_VERBOSE=1 git push origin master 2>&1 | tee push.log
+
+  [ 1 -eq "$(grep -c "Content-Type: application/zip" push.log)" ]
```

Comment:
It looks like this is checking that `application/zip` wasn't found by the grep, but we'd be expecting the opposite

Reply:
Hm, I think that this is checking that the output contained `application/zip` exactly once. According to the manpage, `grep -c` means:

> ```
> -c, --count
>        Only a count of selected lines is written to standard output.
> ```

In which case `-eq` means a numerical comparison. So, we're checking that the number of times we found `Content-Type: application/zip` in the output is exactly one.

Reply:
Ah okay! That makes sense to me. In that case, is the test below this one correct? If we've disabled the content type detection, I'd expect application/zip to not appear in the log.

Reply:
Ack! What a great catch. I wrote a little bit about why this happened in 55ce4393 (t: fix 'git config' invocation in content type tests, 2018-07-31).